CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## neptune-client 0.16.2
```

Comment:
One more thing (I haven't mentioned that before, sorry):
When you create a new version entry in CHANGELOG, please mark it with `[UNRELEASED]`, so everyone can tell at a glance that they can add new entries to it. Admittedly we don't ALWAYS do that, but at least try ;). Example: https://github.com/neptune-ai/neptune-client/pull/825/files